### ISSUE TITLE
linux version can`t log in

### ISSUE THREAD

**Issue by jigaowan** [exilence-ce/exilence-ce#6]
clicking authorization on poe auth site popup window ask open “xdg-open” or cancel, after click open “xdg-open” nothing happened.

**Comment by thepapanoob** [comment]
Hi @jigaowan,

the issue is that you need to add a url handler for exilence-ce for it to work. Ill try to provide an example how to do that and add it to the readme as a note.

**Comment by donkeybanana** [comment]
@jigaowan I just installed for the first time and the auth flow works fine for me.

I use https://github.com/TheAssassin/AppImageLauncher to manage AppImage installs, and it appears to have automatically set up the `.desktop` file for me (the "url handler" papa mentioned).

@thepapanoob I did fork with the intent to add exhaustive instructions but at this point I'd probably just go with recommending https://github.com/TheAssassin/AppImageLauncher for release installs; devs should be able to figure out their local builds easy enough, and the README already covers that.
